builders.ncl

```diff
@@ -34,44 +41,29 @@ let { NickelDerivation, Derivation, .. } = import "contracts.ncl" in
       "%%%
     = NickelDerivation,
 
-  BashShell = {
-    inputs_spec = {
-      bash = {},
-      # Setting the following as a default value conflicts with InputsSpec's
-      # contract default value (`"nixpkgs"`). Maybe InputsSpec shouldn't set a
-      # default value at all?
-      # That being said, `naked_std_env` is an internal compatibility value:
-      # there doesn't seem to be a good reason to make it a default value. It
-      # can still be overridden using `force` if really needed
-      naked_std_env.input = "nickel-nix-internals",
-    },
-    inputs,
-
-    output =
-      {
-        version | default = "0.1",
-
-        # This is required otherwise nix develop will fail with a message that it
-        # only supports bash.
+  NixpkgsPkg
+    | doc m%%%"
```

Comment:
It's customary to use the simplest delimiter for multiline strings, `m%"`. Longer delimiters are mostly used when you have other Nickel mutliline strings or escaped interpolation sequences inside (e.g. when documenting a stdlib function, and you want to include a Nickel example which itself has a mutliline string) and don't want to escape stuff too much. But here, it's just text

```suggestion
    | doc m%"
```